sechub-scan/src/main/java/com/mercedesbenz/sechub/domain/scan/HTMLScanResultReportModelBuilder.java

```diff
@@ -111,6 +105,41 @@ public Map<String, Object> build(ScanSecHubReport report) {
         } else {
             model.put("jobuuid", "none");
         }
+
+        Map<ScanType, ScanTypeCount> scanSummaryMap = new HashMap<>();
+        for (SecHubFinding finding : result.getFindings()) {
+            ScanType scanType = finding.getType();
+            ScanTypeCount scanTypeCount;
+            if (scanSummaryMap.containsKey(scanType)) {
+                scanTypeCount = scanSummaryMap.get(scanType);
+            } else {
+                scanTypeCount = new ScanTypeCount(scanType);
+                scanSummaryMap.put(scanType, scanTypeCount);
+            }
+            incrementScanCount(finding.getSeverity(), scanTypeCount);
+        }
+        List<ScanTypeCount> scanTypeCountList = new ArrayList<>();
```

Comment:
Either use a LinkedList and sort it. Or simply use a [`SortedSet`](https://docs.oracle.com/javase/8/docs/api/java/util/SortedSet.html). SortedSet has the advantage of getting sorted while inserting values.

Reply:
Done